### PR TITLE
Add Aircall (1.1.2) cask

### PR DESCRIPTION
**Pull request by jorge-d**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version. => Sorry I kinda forgot that one, we can squash the pr with the version maybe?

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Hi,

We thought at Aircall.io that it could be great to have a cask recipe, so I just created one,
let me know if everything is good,

Cheers,

### CODE REVIEW

File: Casks/aircall.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'aircall' do
+  version '1.1.2'
+  sha256 '7c0afc31019dfaa6867bc47cdc8b84033b0d98a00eadefb19a78868d23e7c5dc'
+
+  url 'http://electron.aircall.io/download/latest/osx'
+  name 'Aircall'
+  homepage 'https://aircall.io'
+
+  auto_updates true
```
Comment (vitorgalvao) [inline Casks/aircall.rb]: If it auto-updates it has an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md). Chan you figure out what it is?

File: Casks/aircall.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'aircall' do
+  version '1.1.2'
```
Comment (vitorgalvao) [inline Casks/aircall.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by jorge-d** [comment]
Thanks for the review @vitorgalvao, I just made some changes,
Regards

**Comment by vitorgalvao** [comment]
Don’t drop the `auto_updates`. It was fine.

**Comment by jorge-d** [comment]
Ok I plugged it back @vitorgalvao,

however I have a small issue with the appcast checkpoint.
The [electron-release-server](https://github.com/ArekSredzki/electron-release-server/blob/598b010dd8d5dfc7db503bf62422cd1117472bb4/docs/urls.md#update-endpoints) returns a 204 response when version is up to date and `brew cask audit` doesn't like empty responses checksums. So I plugged an old version to the appcast url (which will anyway change checksum when an update is available).
Hope this is ok that way?
Thanks for the fast feedbacks,

**Comment by vitorgalvao** [comment]
Yes, that is fine. Thank you.

**Pull request merged**